commands/python/command/deploy.py

```diff
@@ -101,6 +101,11 @@ def deploy(
         # Values precedence is command < cluster rules < --set-override-values
         # Deploy command values
         values = {
+            "service.globalCertificateDNS": cr.global_certificate_dns,
+            "service.globalCertificateHostname": cr.global_certificate_dns.strip("*."),
+            "service.certificateDNS": cr.certificate_dns,
+            "service.certificateHostname": cr.certificate_dns.strip("*."),
+            "service.globalCertificateArn": cr.get_certificate_arn(get_global=True),
```

Comment:
could really use some suggestions for better names here

Reply:
I'm not a big fan of using "global" prefix for variable identifiers, maybe call it an override instead?

Reply:
It's not really an override, though.
It might be a moot point, as this global namespace is looking more like an anti-pattern that we may not want to encourage.